### DISCUSSION TITLE
where to find Xposed-api.jar

### DISCUSSION THREAD

**Discussion by joobih** [LSPosed/LSPosed#2887] [New Release]
where to find Xposed-api.jar

**Comment by JasonKhew96** [comment]
you can download here https://github.com/rovo89/XposedBridge/tree/gh-pages/de/robv/android/xposed/api
or you can setup maven pointed to `https://api.xposed.info/`,
please refer to https://docs.gradle.org/current/userguide/declaring_repositories.html
